sdk/python/jobs/automl-standalone-jobs/automl-nlp-text-named-entity-recognition-task-distributed-sweeping/automl-nlp-text-ner-task-distributed-with-sweeping.ipynb

```diff
@@ -460,6 +472,59 @@
    ]
   },
   {
+   "attachments": {},
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## 4.3 Manual hyperparameter sweeping for models from Hugging Face (Preview)\n",
+    "Similar to how you can use any model algorithm from Hugging face transformers library for individual runs, you can also include these models to perform a hyperparameter sweep. You can also choose a combination of model algorithms supported supported natively by [AutoML](https://learn.microsoft.com/en-us/azure/machine-learning/how-to-auto-train-nlp-models?view=azureml-api-2&tabs=cli) and model algorithms from [Hugging Face](https://huggingface.co/models?pipeline_tag=token-classification&library=transformers&sort=trending).\n",
```

Comment:
Minor: In Vision notebooks, we have a section called "Individual runs" and it also points to using HF models. The first line here is referring to that. In this case, that section is named differently and we don't have example in this notebook to use hf models with individual runs. So, it might be better to remove the part "Similar to how you can use any model algorithm from Hugging face transformers library for individual runs"

Reply:
Updated the description.